base/services/w32time/register.c

```diff
@@ -236,7 +236,7 @@ SetNtpClientValues(VOID)
         goto done;
     }
 
-    dwValue = 0x00093a80;
+    dwValue = W32TIME_POLL_INTERVAL;
```

Comment:
```suggestion
    dwValue = W32TIME_POLL_INTERVAL; // HACK value (CORE-19281), instead of 7 days.
```

Reply:
I'm not sure why using a 4 hour interval is more hackish than the 7 day "default" windows appears to use.

Reply:
I agree @HBelusca. See my comments below. Thanks.